conversation-graph/convo-graph.bundle.js

```diff
@@ -0,0 +1,254 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
What is this file @Connoropolous ? Is it just like a minified combination of other files? why's it look so weird?